core/spec.md

```diff
@@ -2671,15 +2675,17 @@ The following describes the attributes of Registry model:
   - The singular name of the Resource type e.g. `message` (`RESOURCE`).
   - MUST be non-empty and MUST be a valid attribute name with the exception
     that it MUST NOT exceed 58 characters (not 63).
-  - MUST be unique within the scope of its owning Group type.
+  - MUST be unique (across all Resource plural and singular names) within the
```

Comment:
```suggestion
  - MUST be unique across all Resources (plural and singular names) within the
```

Just for consistency with the above - but if it doesn't work for some reason, please ignore.